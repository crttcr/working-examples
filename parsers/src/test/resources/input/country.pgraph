// Country provides Vertices for all countries
//
//

## vertex.auto.id.generator = STRING_SEQUENCE
## vertex.auto.id.prefix    = xiv.country.iso3

(cyp: COUNTRY { name = "Cyprus"})
(grc: COUNTRY { name = "Greece"})
(gbr: COUNTRY { name = "United Kingdom"})
(lby: COUNTRY { name = "Libya"})
(isr: COUNTRY { name = "Israel"})
(ita: COUNTRY { name = "Italy"})
(fra: COUNTRY { name = "France"})
(usa: COUNTRY { name = "United States"})
