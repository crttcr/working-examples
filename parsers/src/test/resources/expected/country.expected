## vertex.auto.id.generator = STRING_SEQUENCE
## vertex.auto.id.prefix = xiv.country.iso3
VERTEX: cyp
	LABEL: COUNTRY
		PROP name = "Cyprus"
VERTEX: grc
	LABEL: COUNTRY
		PROP name = "Greece"
VERTEX: gbr
	LABEL: COUNTRY
		PROP name = "United Kingdom"
VERTEX: lby
	LABEL: COUNTRY
		PROP name = "Libya"
VERTEX: isr
	LABEL: COUNTRY
		PROP name = "Israel"
VERTEX: ita
	LABEL: COUNTRY
		PROP name = "Italy"
VERTEX: fra
	LABEL: COUNTRY
		PROP name = "France"
VERTEX: usa
	LABEL: COUNTRY
		PROP name = "United States"